python/cvmfs/_common.py

```diff
@@ -30,68 +27,24 @@ def __init__(self):
         Exception.__init__(self, "It seems that cvmfs is not installed on this machine!")
 
 
-class CompressedObject:
-    file_            = None
-    compressed_file_ = None
 
-    def __init__(self, compressed_file):
-        self.compressed_file_ = compressed_file
-        self._decompress()
+class DatabaseObject:
+    _db_handle = None
 
-    def get_compressed_file(self):
-        return self.compressed_file_
-
-    def get_uncompressed_file(self):
-        return self.file_
-
-    def save_to(self, path):
-        shutil.copyfile(self.get_compressed_file().name, path)
-
-    def save_uncompressed_to(self, path):
-        shutil.copyfile(self.get_uncompressed_file().name, path)
-
-    def size_compressed(self):
-        """ Size of the compressed file in bytes """
-        return os.path.getsize(self.compressed_file_.name)
-
-    def size_uncompressed(self):
-        """Size of the uncompressed file in bytes """
-        return os.path.getsize(self.file_.name)
-
-    def _decompress(self):
-        """ Unzip a file to a temporary referenced by self.file_ """
-        self.file_ = tempfile.NamedTemporaryFile('w+b')
-        self.compressed_file_.seek(0)
-        self.file_.write(zlib.decompress(self.compressed_file_.read()))
-        self.file_.flush()
-        self.file_.seek(0)
-        self.compressed_file_.seek(0)
-
-    def _close(self):
-        if self.file_:
-            self.file_.close()
-        if self.compressed_file_:
-            self.compressed_file_.close()
-
-
-
-class DatabaseObject(CompressedObject):
-    db_handle_ = None
-
-    def __init__(self, compressed_db_file):
-        CompressedObject.__init__(self, compressed_db_file)
+    def __init__(self, db_file):
+        self._file = db_file
         self._open_database()
 
     def __del__(self):
-        if self.db_handle_:
-            self.db_handle_.close()
-        self._close()
+        self._file.close()
```

Comment:
There is absolutely no reason. It was my mistake when cleaning up code. I'll re-add it now